content/staging-area.md

```diff
@@ -161,61 +161,134 @@ We give two examples and the instructor can pick one or both:
 
 ## Staging area commands
 
-With the **staging area**, there is one middle point before commits.
+The staging area creates a layer between the working directory and the next
```

Comment:
can this be rephrased to somehow sound less jargony??

Reply:
following up below